portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,227 @@
+							</ul>				
+						</div>
+                        <ul class="actions">
+							<li><a href="#" class="button">Back to top</a></li>
+						</ul>
+                    </section>
+				<!-- Four -->
+                    <section id="four">
+                        <h2>Work Experience</h2>
+                        <div class="col-6 col-12-xsmall">		
+							<ul>
+								<li><h3>Google Taiwan, Software Engineer Practicum Intern</h3></li>
+                                On-site internship under this leading construction company.<br />
+                                     In Pixel phone department and help Pixel
+test suite items (PTS) development in stability, bootloader and security.
+                                <h5> Android 10 Release Celebration</h5>
+                                <div class="box alt">
+								<div class="row gtr-50 gtr-uniform">
+									<div class="col-3"><span class="image fit"><img src="images/fulls/ad10-1.jpg" alt="" /></span></div>
+									<div class="col-3"><span class="image fit"><img src="images/fulls/ad10.jpg" alt="" /></span></div>
+									<div class="col-3"><span class="image fit"><img src="images/fulls/ad10-2.jpg" alt="" /></span></div>
+									<div class="col-3"><span class="image fit"><img src="images/fulls/ad10-3.jpg" alt="" /></span></div>
+								</div>
+							    </div>
+                                <li><h3>NTU COOL, System Administration Intern</h3></li>
+                                    <p><span class="image right"><img src="images/fulls/cool.png" alt="" /></span>
+                                    NTU cool is an university teaching platform for all student in NTU.
+                                    I help NTU Cool develop function and help maintain system for university students. 
+                                    <ol>
+                                        <li>Build automatically update script to help updating NTU cool system and reduce downtime</li>
+                                        <li>Build slack chat bot to message user real time system loading on Grafana server</li>
+                                    </ol>
+                                    </p>
+
+                            </ul>
+                        </div>
+                        <ul class="actions">
+							<li><a href="#" class="button">Back to top</a></li>
+						</ul>
+                    </section>
+					<section id="five">
+                        <h2>Hobby</h2>
+                        I like outdoor activities.<br />
+                        I go jogging or biking almost every day. If it doesn't rain : ) <br /> 
+                        At school, I join table tennis team and girl's volleyball team. <br />
+                        In summer, I also like to go swimming.<br />
+                        When I have free time or go on a vacation, I like to go hiking.<br />
+                        Walking into nature makes me feel relax, I feel alive and release when being in the wild.

```

Comment:
Taking nature walks makes me feel relaxed. I feel alive and free when being in the wild.